Source/Core/Core/PowerPC/Jit64/Jit.h

```diff
@@ -127,7 +129,8 @@ class Jit64 : public JitBase, public QuantizedMemoryRoutines
                             bool duplicate = false);
   void FinalizeDoubleResult(Gen::X64Reg output, const Gen::OpArg& input);
   void HandleNaNs(UGeckoInstruction inst, Gen::X64Reg xmm, Gen::X64Reg clobber,
-                  std::vector<int> inputs);
+                  const std::optional<Gen::OpArg> Ra, const std::optional<Gen::OpArg> Rb,
```

Comment:
if these aren't passed by reference or pointer, then the const does nothing in the declaration

Reply:
Yeah. I iterated on a few different approaches for how to pass the operands, and I think this was a leftover from an iteration where I was using references.

Now that I think about it, the switch away from a vector isn't actually necessary with the current approach... Any opinions on that one?

Reply:
I don't really have that strong an opinion either way. It's nice to not need to heap allocate, though